quantus/evaluation.py

```diff
@@ -72,6 +73,7 @@ def evaluate(
                         y_batch=y_batch,
                         a_batch=a_batch,
                         s_batch=s_batch,
+                        # TODO: what does the method parameter do?
```

Comment:
I believe this is a kwarg for the explain_function. The quantus wrappers around tf_explain and captum take a "method" kwarg. This does not fit with zennit though.

I would suggest we change this a bit. xai_methods should be a dict mapping {explain_method_name: explain_method_kwargs}. The methodname can be used for results, and the method_kwargs to compute explanations. This would fit very well with this PR, and make this xai evaluation a lot more customizable.